docs/data/scheduler/event-calendar/event-calendar.md

```diff
@@ -171,6 +171,7 @@ Available properties:
 - `toggleWeekendVisibility`: show/hide the menu item that toggles weekend visibility.
 - `toggleWeekNumberVisibility`: show/hide the menu item that toggles week number visibility.
 - `toggleAmpm`: show/hide the menu item that toggles 12/24‑hour time format.
+- `toggleEmptyDaysInAgenda`: show/hide the menu item that toggles empty days visibility in the Agenda view.
```

Comment:
Should we detail that `empty days` means days without events? 🤔 Not sure, mostly a nit.